geometry_msgs/PoseWithCovarianceStamped robot_pose
---